packages/site/src/types/content.ts

```diff
@@ -12,7 +12,7 @@ export interface ContentExport {
     element: unknown;
     defaultProps: Record<
       string,
-      string | boolean | JSX.Element | [] | (() => void)
+      string | boolean | number | JSX.Element | [] | (() => void)
```

Comment:
Is this an okay change? I needed it or something similar for this example :)

Reply:
Makes sense for now at least!